src/ansys/fluent/core/fluent_connection.py

```diff
@@ -407,24 +437,22 @@ def register_finalizer_cb(self, cb):
         """Register a callback to run with the finalizer."""
         self.finalizer_cbs.append(cb)
 
-    def create_service(self, service, add_arg=None):
+    def create_service(self, service, args=()):
```

Comment:
For additional optional arguments, the signature would be `def create_service(self, service, *args):`